src/containers/Results/Result/index.js

```diff
@@ -37,7 +37,18 @@ const Result = ({ result, query }) => {
             className="link result__title"
             to={routes.experiments(result.accession_code, {
               ref: 'search',
-              result
+              result: {
```

Comment:
So here some attributes are removed from `result`, and a new `samples` attribute is added?

Reply:
Yes, basically to avoid adding too much information in the url state. 

The samples attribute will be removed with https://github.com/AlexsLemonade/refinebio-frontend/pull/433, once that's merged the API won't send any detailed sample information. I added a default value here to avoid changing the experiments page here since that's updated on #433